source/developers-guide/plugin-configuration/index.md

```diff
@@ -618,12 +298,12 @@ public function onPostDispatch(Enlight_Event_EventArgs $arguments)
  
     $view->addTemplateDir($this->Path() . 'Views/');
 }
-```
 
-Within the `postDispatch()` method we create an instance of the configuration. Afterwards we test for the just created __show__ attribute. If
-this attribute is empty the method ends and the plugin will not be executed or shown any further. If __show__ is set the template is loaded and 
-assigned via __smarty__.
 ```
+
+Within the onPostDispatch() method in a subscriber we create an instance of the configuration. Afterwards we test for the just created show attribute. If this attribute is empty the method ends and the plugin will not be executed or shown any further. If show is set the template is loaded and assigned via smarty.
```

Comment:
the `show` attribute is nowhere created in the article

Reply:
d